### PR TITLE
Modifications for Max Planck Society faculties

### PR DESCRIPTION
**Pull request by jcpassy**
This PR updates profiles of faculties belonging to the Max Planck Society:
* Peter Dayan's profile added
* IS note added to Moritz Hardt's existing profile

### CODE REVIEW

File: csrankings-p.csv [hunk 1]
```diff
@@ -618,6 +618,7 @@ Peter D. Bennett,University of Bristol,http://www.peteinfo.com,t_vvNfcAAAAJ
 Peter D. Grünwald,CWI,http://homepages.cwi.nl/~pdg,wItp6h8AAAAJ
 Peter Damaschke,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/ptr.aspx,NOSCHOLARPAGE
 Peter Danielis,University of Rostock,https://www.vhr.uni-rostock.de/team/vertretungsprofessur/dr-ing-habil-peter-danielis-vertretungsprofessur,XxRH9IoAAAAJ
+Peter Dayan,Max Planck Society,https://www.kyb.tuebingen.mpg.de/computational-neuroscience,_30hSU8AAAAJ
```
Comment (emeryberger) [inline csrankings-p.csv:621]: Not in CS, please clarify.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ database. Use a text editor like emacs or NotePad instead._
 
 **Inclusion criteria**
 
-- [ ] Make sure that any faculty you add meet the inclusion
+- [X] Make sure that any faculty you add meet the inclusion
```
Comment (emeryberger) [inline CONTRIBUTING.md]: This whole file needs to not be present in the PR. DO NOT COMMIT A DELETION - please just remove it from the PR or open a new PR without this change.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-p.csv (hunk 1 above)
Comment (jcpassy) [reply csrankings-p.csv:621]: Hi @emeryberger,
Although mainly based in the MPI for Biological Cybernetics, Peter Dayan also works in fields related to computer science (ML, AI). He is also affiliated to the Department of Computer Science at the University of Tübingen, and is part of the whole AI ecosystems in Tübingen (MPI Campus, Cyber Valley...).
More details here:
https://www.mpg.de/12309370/biological-cybernetics-dayan

File: CONTRIBUTING.md (hunk 2 above)
Comment (jcpassy) [reply CONTRIBUTING.md]: Done, file has been removed.
Thanks!

File: csrankings-p.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings-p.csv:621]: This doesn't explain how this faculty member meets the inclusion criteria (able to solely advise a PhD _in Computer Science_, for instance).

**Pull request merged**